packages/jsapi-utils/src/TableUtils.ts

```diff
@@ -1825,6 +1887,11 @@ export class TableUtils {
         dh.LongWrapper.ofString(TableUtils.removeCommas(value))
       );
     }
+    if (TableUtils.isDateType(type)) {
+      return dh.FilterValue.ofNumber(
+        DateUtils.trimDateTimeStringTimeZone(value) as unknown as number
```

Comment:
Does this actually work? I wouldn't expect a date `String` to work being passed into `ofNumber` (just looking at the code).

Reply:
It does work, but yeah that does look strange. I had based it on the call to `makeQuickDateFilterWithOperation` inside of `makeQuickDateFilter` which passes a string.

Reply:
Oh actually, I think I had actually based this on the fact that this if check wasn't there before and the fallthrough case is just passing to `dh.FilterValue.ofNumber`. The code I added was simply to trim the date string and no longer strip commas

Reply:
hmm, Looks like maybe this function doesn't actually get called on Date columns which would explain why it works

Reply:
I removed that block of code since it's never actually hit